actors.md

```diff
@@ -79,16 +79,20 @@ func Exec(code Cid, params []byte) Address {
 
 	// This generates a unique address for this actor that is stable across message
 	// reordering
-	addr := VM.ComputeActorAddress()
+	// TODO: where do `creator` and `nonce` come from?
+	addr := VM.ComputeActorAddress(creator, nonce)
```

Comment:
is `creator` the same as `message.from`?

Reply:
creator is `msg.From`, and nonce is `msg.Nonce`.